Python/traceback.c

```diff
@@ -385,20 +385,20 @@ _Py_FindSourceFile(PyObject *filename, char* namebuf, size_t namelen, PyObject *
 int
 _Py_WriteIndent(int indent, PyObject *f)
 {
-    int err = 0;
     char buf[11] = "          ";
     assert(strlen(buf) == 10);
     while (indent > 0) {
         if (indent < 10) {
             buf[indent] = '\0';
         }
-        err = PyFile_WriteString(buf, f);
-        if (err == -1)
-            goto done;
+        if (PyFile_WriteString(buf, f) < 0) {
+            goto error;
+        }
         indent -= 10;
     }
-done:
-    return err;
+    return 0;
+error:
+    return -1;
```

Comment:
I adjust my previous review comment! :) For cases like this (no clean-up needed at the `error` label), save a jump and just `return -1` (instead of `goto error` => `return -1`). Perhaps the compiler will optimise the jump away, but it makes for more compact code.